src/goto-symex/symex_other.cpp

```diff
@@ -233,8 +235,8 @@ void goto_symext::symex_other(
     // the right-hand side is an equality over the arrays, if their types match;
     // if the types don't match the result trivially is false
     DATA_INVARIANT(
-      code.operands().size() == 3,
-      "array_equal expected to take three arguments");
+  		code.operands().size() == 3,
```

Comment:
And again here - the original invariant had a more specific description. And indentation again...